core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java

```diff
@@ -205,7 +207,12 @@ private Object replaceVariables(String paramString, DocumentContext documentCont
                     }
 
                 } else {
-                    convertedValues[i] = documentContext.read(paramPath);
+                    try {
+                        convertedValues[i] = documentContext.read(paramPath);
+                    }catch (Exception e) {
+                        logger.warn("documentContext read paramPath: {} error", paramPath);
```

Comment:
Please also print out the exception stack trace in the log statement.
logger.warn("Error reading documentContext for paramPath: {}. Exception: {}", paramPath, e);

Reply:
ok